citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -160,11 +128,82 @@ $(function() {
 	});
 });
 
+function editAuthor(){
+	var authorItem = $(this).parent();
+	$("#firstNameAuthor").val(authorItem.attr("data-author-firstname"));
+	$("#lastNameAuthor").val(authorItem.attr("data-author-lastname"));
+	$("#uriAuthor").val(authorItem.attr("data-author-uri"));
+	$("#idAuthor").attr("data-author-id", authorItem.attr("id"));
+	
+	authorItem.children("span").each(function(idx, elem){
+		var affInput = $("#affiliationTemplate").clone();
+		affInput.removeAttr("id");
+		affInput.attr("class", affInput.attr("class")+" aff-info");
+		affInput.find("input").attr("data-affiliation-name", $(elem).data("affiliationName"));
+		affInput.find("input").attr("data-affiliation-id", $(elem).data("affiliationId"));
+		affInput.find("input").val($(elem).data("affiliationName"));
+		$("#authorModal #affiliations").append(affInput);
+	});
+	$("#addAuthorButton").text("Update Author");
+	$("#authorModal").modal('show');
+}
+
+function saveAuthor() {
+	if($("#idAuthor").attr("data-author-id")){
+		authorSpan = $('#'+$("#idAuthor").attr("data-author-id"));
+	} else {
+		$("#idAuthor").attr("data-author-id", $("#authorList").length);
+		authorSpan = $('<span id="author'+$("#idAuthor").attr("data-author-id")+'">');
+	}
+	authorSpan.html("");
+	var firstname = $("#firstNameAuthor").val();
+	var lastname = $("#lastNameAuthor").val();
+	var uri = $("#uriAuthor").val();
+	var localAuthority = $("#uriAuthorLocalId").val();
+	authorSpan.attr("class", "label label-primary author-item");
+	authorSpan.attr("data-author-firstname", firstname);
+	authorSpan.attr("data-author-lastname", lastname);
+	authorSpan.attr("data-author-uri", uri);
+	authorSpan.attr("data-author-authority-id", localAuthority);
+	
+	var affiliationsList = [];
+	$("#affiliations").children().each(function(idx, elem){
+		var input = $(elem).find("input");
+		if(input.val().length!=0){
+			var affSpan = $("<span>");
+			affSpan.attr("data-affiliation-id", input.attr("data-affiliation-id"));
+			affSpan.attr("data-affiliation-name", input.val());
+			affSpan.val(input.val());
```

Comment:
isn't affSpan a span element? what does calling val() accomplish? form jquery documentation: "This method is typically used to set the values of form fields."